transactions.go

```diff
@@ -600,6 +600,11 @@ type FaintTransaction struct {
 }
 
 func (t FaintTransaction) Mutate(b *Battle) {
+	if b.ruleset&BattleRuleFaint == 0 {
+		pkmn := b.getPokemon(t.Target)
```

Comment:
Log a message to `blog` here that mentions that fainting is disabled, and the pokemon has been restored to max heath.